klasses/bootcamp_admissions_client.py

```diff
@@ -76,6 +77,21 @@ def fetch_legacy_admissions(user_email):
         return {}
 
 
+def fetch_fluidreview_klass_keys(fluid_user_id):
+    """
+    Collect all the unique award ids (== klass ids) from WebhookRequests by user email
+
+    Args:
+        fluid_user_id(int): FluidReview id for a user
+
+    Returns:
+        list: FluidReview award_ids for a user
+    """
+    return list(
+        WebhookRequest.objects.filter(user_id=fluid_user_id).distinct('award_id').values_list('award_id', flat=True)
```

Comment:
You need to also filter on the status of `WebhookRequest`

Reply:
Added the filter and a test